README.md

```diff
@@ -1,3 +1,203 @@
 # product-details
 
-legoland
\ No newline at end of file
+## CRUD Operations for product-details
+
+### Get a product
+* GET `/product/:pid`
```

Comment:
I saw the documentation about the endpoint and they usually recommend for us to using the plural.